src/sentry/models/projectcodeowners.py

```diff
@@ -100,6 +102,10 @@ def update_schema(self, raw: str | None = None) -> None:
         if not self.raw:
             return
 
+        if len(self.raw) > MAX_RAW_LENGTH:
+            logger.error({"raw": f"Raw needs to be <= {MAX_RAW_LENGTH} characters in length"})
```

Comment:
Notice that `logger.error` will create an error on Sentry which is okay but wanted to let you know.

Reply:
Yeah,  maybe this isn't ideal. Would a warning be better?